src/main.ts

```diff
@@ -12,7 +11,6 @@ import './vuegtag';
 
 Vue.config.productionTip = false;
 Vue.use(AsyncComputed);
```

Comment:
This can be removed along with the entry in `package.json` as it's moot with the switch to  composition API.